gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BundleTaskExtension.java

```diff
@@ -335,8 +362,6 @@ public void setSourceSet(SourceSet sourceSet) {
 			.getTasks()
 			.named(sourceSet.getCompileJavaTaskName(), AbstractCompile.class)
 			.flatMap(AbstractCompile::getDestinationDirectory));
-
-		jarLibraryElements(getTask(), sourceSet.getCompileClasspathConfigurationName());
```

Comment:
I think we should be trying to get rid of `jarLibraryElements` entirely since it affects project-level resolution and can cause the same problem to happen elsewhere.

This is still being called from here (in two places):
https://github.com/bndtools/bnd/blob/6e8bbb257fe83539f6158628be828fef5c56dce3/gradle-plugins/biz.aQute.bnd.gradle/src/main/java/aQute/bnd/gradle/BndPlugin.java#L258

I think this means there's a similar problem, but it's much less likely to be a real problem because we always configure the compile tasks.  With the ArtifactView being used for the classpath, I'm not sure if these calls are necessary now.

Reply:
I tried this and it seems to work.  I made a separate PR here: https://github.com/bndtools/bnd/pull/5702 in case there is another reason it should remain.